### PR TITLE
feat(radar): add recent AI squad findings

### PR DESCRIPTION
**Pull request by anandaroop**
Adds to the Tech Radar [some conclusions](https://docs.google.com/spreadsheets/d/1xWJks1wbe8qV44cr636MUaU0rZyAkNYNob3IHigCZY4/edit?usp=sharing) reached by the AI Innovation Squad.

(Radar change is in db2993bcce91047ba01dc6bae95022f87be45260)

### CODE REVIEW

File: playbooks/technology_radar/artsy-tech-radar.csv [hunk 1]
```diff
@@ -36,19 +35,25 @@ Sidekiq,adopt,tools,FALSE,
...
+Vercel AI SDK,assess,languages & frameworks,TRUE,"Solid LLM abstractions in Typescript for text and data generation, with easily swappable models"
 Apollo Server,assess,platforms,TRUE,
 APIs as a product,assess,techniques,TRUE,
+Embeddings,assess,techniques,TRUE,"Dense vector mathematical representation of content, suitable for semantic search"
 GraphQL batching,assess,techniques,TRUE,
 Apollo Platform,assess,tools,TRUE,
 Datadog Synthetics,assess,tools,TRUE,
 HashiCorp Vault,assess,tools,TRUE,
 Helm,assess,tools,TRUE,
+Large Language Models,assess,tools,TRUE,Deep learning models for text generation and embeddings
```
Comment (anandaroop) [inline playbooks/technology_radar/artsy-tech-radar.csv:55]: @joeyAghion I wasn't sure about how to use the `isNew` column. So I set all these new entries to `TRUE` — was there something else intended for this column?

File: playbooks/README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ practice should be incorporated, submit a PR!
 | [How to run a Knowledge Share](/playbooks/running-knowledge-share.md#readme) | How to handle the people process for Knowledge Shares |
 | [System Criticality](/playbooks/system-criticality.md#readme) | A framework for treating systems differently according to how critical they are |
 | [Taming notifications](/playbooks/taming-notifications.md#readme) | How to set up notifications so that they leave you alone outside of working hours |
-| [Technical planning](/playbooks/technical-planning.md#readme) | How to write technical plans at Artsy |
+| [Technology choices](/playbooks/technical-planning.md#readme) | How to write technical plans at Artsy |
```
Comment (joeyAghion) [inline playbooks/README.md:49]: Error? (The tech choices doc is already listed below.)

File: playbooks/technology_radar/artsy-tech-radar.csv (hunk 1 above)
Comment (joeyAghion) [reply playbooks/technology_radar/artsy-tech-radar.csv:55]: That's a Thoughtworks attribute that just renders the blip a little more visibly. I think the additions can all rightly be considered "new." We could even stop calling some old ones new, but that's for a separate update.

**Review by joeyAghion — APPROVED** [review]
Really appreciate this effort to record the team's learnings!

File: playbooks/README.md (hunk 2 above)
Comment (anandaroop) [reply playbooks/README.md:49]: Strange! Not sure how the linting would have touched that, but I'll push up a fix.

Comment (anandaroop) [reply playbooks/README.md:49]: Ah, that must have been inferred from this frontmatter when the linter ran. I'll update that…

https://github.com/artsy/README/blob/ca1c2357b073863512e7deff57f669713404788f/playbooks/technical-planning.md?plain=1#L1-L4

**Comment by anandaroop** [comment]
Merging!

**Pull request merged**